server/protocols/protocols-smtp/src/test/java/org/apache/james/smtpserver/ValidRcptHandlerTest.java

```diff
@@ -26,47 +27,54 @@
 import org.apache.james.core.Domain;
 import org.apache.james.core.MailAddress;
 import org.apache.james.dnsservice.api.DNSService;
-import org.apache.james.domainlist.api.DomainList;
 import org.apache.james.domainlist.memory.MemoryDomainList;
-import org.apache.james.protocols.api.ProtocolSession.State;
 import org.apache.james.protocols.smtp.SMTPSession;
 import org.apache.james.protocols.smtp.hook.HookReturnCode;
 import org.apache.james.protocols.smtp.utils.BaseFakeSMTPSession;
-import org.apache.james.rrt.api.RecipientRewriteTable;
-import org.apache.james.rrt.api.RecipientRewriteTableException;
 import org.apache.james.rrt.memory.MemoryRecipientRewriteTable;
 import org.apache.james.smtpserver.fastfail.ValidRcptHandler;
 import org.apache.james.user.api.UsersRepository;
 import org.apache.james.user.memory.MemoryUsersRepository;
 import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
 
 public class ValidRcptHandlerTest {
-
     private static final String VALID_USER = "postmaster";
     private static final String INVALID_USER = "invalid";
     private static final String USER1 = "user1";
     private static final String USER2 = "user2";
-    UsersRepository users;
-    ValidRcptHandler handler;
+    private static final String PASSWORD = "xxx";
+    private static final boolean RELAYING_ALLOWED = true;
+
+    private ValidRcptHandler handler;
+    private MemoryRecipientRewriteTable memoryRecipientRewriteTable;
+    private MailAddress validUserEmail;
+    private MailAddress user1mail;
+    private MailAddress invalidUserEmail;
 
     @Before
     public void setUp() throws Exception {
-
-        users = MemoryUsersRepository.withoutVirtualHosting();
-        users.addUser(VALID_USER, "xxx");
+        UsersRepository users = MemoryUsersRepository.withoutVirtualHosting();
+        users.addUser(VALID_USER, PASSWORD);
 
         MemoryDomainList memoryDomainList = new MemoryDomainList(mock(DNSService.class));
         memoryDomainList.setDefaultDomain(Domain.LOCALHOST);
 
+        memoryRecipientRewriteTable = new MemoryRecipientRewriteTable();
+        memoryRecipientRewriteTable.setDomainList(memoryDomainList);
+
         handler = new ValidRcptHandler();
         handler.setUsersRepository(users);
-        handler.setRecipientRewriteTable(setUpRecipientRewriteTable(memoryDomainList));
+        handler.setRecipientRewriteTable(memoryRecipientRewriteTable);
         handler.setDomainList(memoryDomainList);
-    }
 
-    private SMTPSession setupMockedSMTPSession(final boolean relayingAllowed) {
+        validUserEmail = new MailAddress(VALID_USER + "@localhost");
+        user1mail = new MailAddress(USER1 + "@localhost");
```

Comment:
user1 is valid ? it looks like the naming convention doesn't fit more than 2 users (valid and invalid)